src/components/attendee/errors/BlankPage.module.scss

```diff
@@ -0,0 +1,27 @@
+@use "scss/attendee/font";
+@use "scss/attendee/theme";
+
+.container {
+  height: 100vh;
```

Comment:
[Question] is there any advantage using `vh`/`vw` in comparison to percents `%` in this case?

Reply:
`vh` and `vw` do work as intended: the screen (viewport) width and height. 
The `%` is (semantically) relative to parent so might not always be the same, thus it's a clearer intention to use `vh` and `vw`.

NOTE: there are more viewport units on the way in CSS that will make them even more useful than `%`.